tests/model_selection/dask_searchcv/test_model_selection.py

```diff
@@ -975,21 +976,35 @@ def test_mock_with_fit_param_raises():
         clf.fit(X, y)
 
 
-@pytest.mark.parametrize("metric,greater_is_better,needs_proba,sample_wt", [
-    (accuracy_score, True, False, [1, 999999, 1, 999999]),
-    (accuracy_score, True, False, [100000, 200000, 100000, 200000]),
-    (accuracy_score, True, False, [100000, 100000, 100000, 100000]),
-    (accuracy_score, True, False, [200000, 100000, 200000, 100000]),
-    (accuracy_score, True, False, [999999, 1, 999999, 1]),
-    (accuracy_score, True, False, [2000000, 1000000, 1, 999999]),
-    (log_loss, False, True, [2500000, 500000, 200000, 100000]),
-    (brier_score_loss, False, True, [2500000, 500000, 200000, 100000]),
+IMP_WT_LOG_REG_PARAMS = {
+    "solver":         ["lbfgs"],
+    "penalty":        ["l2"],
+    "tol":            [1e-6],
+    "max_iter":       [10000],
+    "fit_intercept":  [False],
+    "random_state":   [15432]
+}
+LOG_REG_PIPELINE = Pipeline([("clf", LogisticRegression())])
+IMP_WT_PIPE_LOG_REG_PARAMS = {"clf__"+k: v for k, v in IMP_WT_LOG_REG_PARAMS.items()}
+
+@pytest.mark.parametrize("metric,greater_is_better,needs_proba,estimator,estimator_params,sample_wt", [
+    (accuracy_score, True, False, LogisticRegression(), IMP_WT_LOG_REG_PARAMS, [1, 999999, 1, 999999]),
+    (accuracy_score, True, False, LogisticRegression(), IMP_WT_LOG_REG_PARAMS, [100000, 200000, 100000, 200000]),
+    (accuracy_score, True, False, LogisticRegression(), IMP_WT_LOG_REG_PARAMS, [100000, 100000, 100000, 100000]),
+    (accuracy_score, True, False, LogisticRegression(), IMP_WT_LOG_REG_PARAMS, [200000, 100000, 200000, 100000]),
+    (accuracy_score, True, False, LogisticRegression(), IMP_WT_LOG_REG_PARAMS, [999999, 1, 999999, 1]),
+    (accuracy_score, True, False, LogisticRegression(), IMP_WT_LOG_REG_PARAMS, [2000000, 1000000, 1, 999999]),
+    (accuracy_score, True, False, LOG_REG_PIPELINE, IMP_WT_PIPE_LOG_REG_PARAMS, [2000000, 1000000, 1, 999999]),
+    (log_loss, False, True, LogisticRegression(), IMP_WT_LOG_REG_PARAMS, [2500000, 500000, 200000, 100000]),
+    (brier_score_loss, False, True, LogisticRegression(), IMP_WT_LOG_REG_PARAMS, [2500000, 500000, 200000, 100000]),
 ])
 def test_sample_weight_cross_validation(
```

Comment:
Looks like this is zefr test case? If so can we leave a comment here so that it can be easily found later?

Reply:
will do